api-client/src/i18n/DateFormatter.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import type Moment from 'moment'
+import moment from 'moment'
+
+export type FormatFunctionType = (date: Moment, options: {| format?: string |}) => string
+
+class DateFormatter {
+  _fallbackFormat: string | void
+  _defaultLocale: string
+
+  constructor (fallbackFormat: string | void, defaultLocale: string) {
+    this._fallbackFormat = fallbackFormat
+    this._defaultLocale = defaultLocale
+  }
+
+  format: FormatFunctionType = (date: Moment, options: {| format?: string |}) => {
+    const format = options.format || this._fallbackFormat
+    // TODO IGAPP-399: Uncomment again and use locale instead of hardcoded 'en'
```

Comment:
So timestamps are using english format in the webapp for all languages now as well?

Reply:
For now yes, I think it is not worth to add a workaround :/

Reply:
Yeah, think so as well. Was just asking :D